doc/quickstart/index.rst

```diff
@@ -1,31 +1,113 @@
 Start Here
 ==========
 
-CCF releases (``ccf.tar.gz``) are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases>`_. Once downloaded, the extracted install directory can be copied to a long-lived path, e.g. ``/opt/ccf-install``.
+Container
+---------
 
-The install directory contains (under ``getting_started/``) the Azure and Ansible scripts required to setup the CCF environment. See :ref:`how to setup the CCF environment here <quickstart/requirements:Azure Confidential Compute>`.
+The quickest way to get started building CCF applications is to use the :ref:`developers/build_app:Build Container`.
 
-.. note::
+.. code-block:: bash
+
+    sudo docker run -it ccfciteam/ccf-app-ci:0.11
```

Comment:
Should we say `:latest` for docker versions, rather than keeping these docs in-sync manually?

Reply:
We don't have a latest (yet), but that is an excellent point.

Reply:
Done.